values.yaml

```
@@ -63,6 +63,24 @@ environmentSecrets: []
   #     name: retool-github-app-private-key
   #     key: private-key
 
+# Optionally specify environmental variables. Useful for variables that are not key-value, as env: {} above requires.
+# Can also include environment secrets here instead of in environmentSecrets
+extraEnvs: []
+  #   - name: SCIM_AUTH_TOKEN
+  #     valueFrom:
+  #       secretKeyRef:
+  #         name: retool-scim-auth-token
+  #         key: auth-token
+  #   - name: GITHUB_APP_PRIVATE_KEY
+  #     valueFrom:
+  #       secretKeyRef:
+  #         name: retool-github-app-private-key
+  #         key: private-key
+  #   - name: NAME
```

Nitpick:
```suggestion
  #   - name: POD_HOST_IP
```

That's not the exact name we'd use, but it better clarifies the example I think.